### PR TITLE
Update Liblouis to 3.27.0

### PR DESCRIPTION
**Pull request by codeofdusk**
### Link to issue number:
None

### Summary of the issue:
Liblouis 3.27.0 has been released.

### Description of how this pull request fixes the issue:
Update Liblouis to 3.27.0, including new Braille tables.

### Testing strategy:
Verified that Braille functions without errors and that new tables are selectable.

### Known issues with pull request:
None known

### Change log entries:
== Changes ==
- Updated LibLouis braille translator to 3.27.0.
    - Added new Thai and Romanian Braille tables.

### Code Review Checklist:

- [x] Pull Request description:
  - description is up to date
  - change log entries
- [x] Testing:
  - Unit tests
  - System (end to end) tests
  - Manual testing
- [x] API is compatible with existing add-ons.
- [x] Documentation:
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
- [x] UX of all users considered:
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English
- [x] Security precautions taken.

### CODE REVIEW

File: source/brailleTables.py [hunk 1]
```diff
@@ -452,6 +452,9 @@ def listTables():
 addTable("pu-in-g1.utb", _("Punjabi grade 1"))
 # Translators: The name of a braille table displayed in the
 # braille settings dialog.
+addTable("ro-g0.utb", _("Romanian 6 dot"))
+# Translators: The name of a braille table displayed in the
+# braille settings dialog.
 addTable("ro.ctb", _("Romanian"))
```
Comment (leonardder) [inline source/brailleTables.py:458]: Should this one be renamed in the GUI?

Comment (codeofdusk) [reply source/brailleTables.py:458]: I'm not sure.

Comment (leonardder) [reply source/brailleTables.py:458]: Having a 6 dot and a nothing is a bit confusing.

Comment (codeofdusk) [reply source/brailleTables.py:458]: Hmmm, how about Romanian (legacy)?

Comment (seanbudd) [reply source/brailleTables.py:458]: Perhaps it's worth asking our romanian translators: https://github.com/nvaccess/mrconfig/blob/master/scripts/addressData.py#L358

Comment (seanbudd) [reply source/brailleTables.py:458]: Have we heard anything back yet?

Comment (seanbudd) [reply source/brailleTables.py:458]: I'm assuming we haven't heard anything back yet. In absence of a response, lets merge and fix the strings later if required?

Comment (codeofdusk) [reply source/brailleTables.py:458]: No response yet, sounds reasonable to me

**Review by seanbudd — APPROVED** [review]

**Pull request merged**